src/GitHubRepoManager/Functions/Public/Set-GitHubRepoVulnerabilitySetting.ps1

```diff
@@ -23,21 +25,29 @@ function Set-GitHubRepoVulnerabilitySetting {
         [Parameter(Mandatory = $true)]
         [String]$GitHubOrg,
         [Parameter(Mandatory = $true)]
-        [String]$RepoPrefix
+        [String]$RepoPrefix,
+        [Parameter(Mandatory = $false)]
+        [String]$ExcludedRepoPrefix
     )
 
-    $Uri = "/orgs/$GitHubOrg/repos?type=all"
-    Write-Verbose "Searching org $GitHubOrg"
-    $Repos = Invoke-GitHubRestMethod -Method GET -Uri $uri
+    $Repos = Get-GitHubRepo -GitHubOrg $GitHubOrg | Where-Object { $_.name -match "^$RepoPrefix.+" -and $_.isArchived -eq $false }
     Write-Verbose -Message "Got $($Repos.Length) repositories"
 
-    $MatchedRepos = $Repos | Where-Object {$_.name -match "^$RepoPrefix.+" -and $_.archived -eq $false}
+    if ($ExcludedRepoPrefix) {
+        $Repos = $Repos | Where-Object { $_.name -notmatch "^$ExcludedRepoPrefix.+" }
+        Write-Verbose -Message "With excluded repository prefix, got $($Repos.Length) repositories"
+    }
 
-    if (!$MatchedRepos) {
+    if (!$Repos) {
         Write-Host "No unarchived repositories found in org $GitHubOrg that match prefix $RepoPrefix"
+        if ($ExcludedRepoPrefix) {
+            Write-Host "with excluded prefix $ExcludedRepoPrefix"
+        }
     }
-    else{
-        foreach ($Repo in $MatchedRepos) {
+    else {
+        Write-Host "In else block"
```

Comment:
Change message to "Unarchived repositories found in org $GitHubOrg that match prefix $RepoPrefix"